src/components/BlockchainsStack/BlockchainsStack.module.scss

```diff
@@ -142,3 +149,15 @@
   font-size: var(--text-md);
```

Comment:
Not sure if this piece here is related to this is, but, as we now have the new typography implemented, we could use eyebrow-sm here so the font doesn't surpass the space inside the circle when there are 2 numbers: 
![image](https://github.com/user-attachments/assets/501d34ef-4e7b-4e6f-949d-d69bc1b19b2a)

Reply:
The way it is now, it looks like it is not centered:
![image](https://github.com/user-attachments/assets/58cfd3b2-221f-4bdb-8d09-6c1ad0820a69)